Rakefile

```diff
@@ -110,6 +111,11 @@ def release
 
   bucket.create_file("build/tuist.zip", "latest/tuist.zip").acl.public!
   bucket.create_file("build/tuistenv.zip", "latest/tuistenv.zip").acl.public!
+  Dir.mktmpdir do |tmp_dir|
+    version_path = File.join(tmp_dir, "version")
+    File.write(version_path, version)
+    bucket.create_file(version_path, "latest/version").acl.public!
```

Comment:
Thanks to having this file, we can get the latest version by just reading that resource, no need to hit the GitHub API anymore.